generator/internal/descriptor_utils.cc

```diff
@@ -561,14 +561,13 @@ std::string FormatAdditionalPbHeaderPaths(VarsDictionary& vars) {
 std::string GetEffectiveServiceName(VarsDictionary const& vars,
                                     absl::string_view name) {
   auto service_name_mappings = vars.find("service_name_mappings");
-  if (service_name_mappings == vars.end()) {
-    return std::string(name.data(), name.size());
-  }
-  for (absl::string_view arg :
-       absl::StrSplit(service_name_mappings->second, ',')) {
-    std::pair<absl::string_view, absl::string_view> p =
-        absl::StrSplit(arg, absl::MaxSplits('=', 1));
-    if (p.first == name) return std::string(p.second.data(), p.second.size());
+  if (service_name_mappings != vars.end()) {
```

Comment:
You might consider the same below, just for symmetry, but it's less compelling given the simple nature of the return value.

Reply:
Yeah that's what I was thinking too... 

Gonna leave it as is.

Reply:
OK.

For a parting thought, also consider how both functions are the same modulo the variable they look up, and how they return the "not found" value.  Perhaps they should be unified?